Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:carbon as build
+
+# Create app directory
+WORKDIR /usr/src/pika
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY pika/package*.json ./
+
+# If you are building your code for development
+# RUN npm install
+RUN npm install --only=production
+
+# Bundle app source
+COPY pika/ .
```

Comment:
Wouldn't is be better copying this first to prevent this from overwriting newly installed packages? On the other hand this could be exactly the expected behavior.